docs/setup_xcode.md

```diff
@@ -234,280 +159,232 @@ $ tree
 
-int count(vector<double> v) {
-  assert(false);
-}
+Select all the starter files (<kbd>Command</kbd> + <kbd>a</kbd>).  Do *not* select "Copy items if needed".  Click "Add".
 
-double sum(vector<double> v) {
-  assert(false);
-}
+<img src="images/xcode100.png" width="384px" />
 
-double mean(vector<double> v) {
-  assert(false);
-}
+You will now see your files in the sidebar.
 
-double median(vector<double> v) {
-  assert(false);
-}
+<img src="images/xcode110.png" width="256px" />
 
-double mode(vector<double> v) {
-  assert(false);
-}
+#### Rename files
+If you need to rename any files, use Xcode, not the command line or Finder.  In EECS 280, you'll need to rename any files that end in `.starter`.
 
-double min(vector<double> v) {
-  assert(false);
-}
+Select a file and press <kbd>Enter</kbd>.  Change the file name.  In EECS 280, you'll do this to any file that ends in `.starter`.
 
-double max(vector<double> v) {
-  assert(false);
-}
+| <img src="images/xcode120.png" width="192px" /> | <img src="images/xcode125.png" width="192px" /> |
 
-double stdev(vector<double> v) {
-  assert(false);
-}
-
-double percentile(vector<double> v, double p) {
-  assert(false);
-}
-```
-{: data-title="stats.cpp" }
-
-### Project 1 `main.cpp`
-Start your `main.cpp` like this.  All it does so far is "hello world".  We'll include a few libraries that will be useful later.
-```c++
-// main.cpp
-// Project UID 5366c7e2b77742d5b2142097e51561a5
+## Compile and Run
+An Xcode target compiles (builds) one executable.
 
-#include "stats.h"
-#include "p1_library.h"
-#include <iostream>
-using namespace std;
-
-int main() {
-  cout << "hello from main!\n";
-}
-```
-{: data-title="main.cpp" }
-
-
-# Compile
-An Xcode target compiles (builds) one executable.  One executable should have exactly one `main()` function.  Three of our project 1 files have `main()` functions.
-
-| Project 1 Target | File with `main()` | Other `.cpp` Build Sources |
-| ------ | --------------- |
-| `stats_tests.exe` | `stats_tests.cpp` | `stats.cpp`, `p1_library.cpp` |
-| `stats_public_test.exe` | `stats_public_test.cpp` | `stats.cpp`, `p1_library.cpp` |
-| `main.exe` | `main.cpp` | `stats.cpp`, `p1_library.cpp` |
+We need to avoid multiple `main()` functions by including the right files in our target.  Click "p1-stats" in the side bar, then "p1-stats" under TARGETS.  Click "Build Phases", the "Compile Sources".  Include the files needed to compile the main program.  Remove any other files by selecting them and clicking on the "–" button.
 
+<img src="images/xcode130.png" width="768px" />
 
-## Compile at the command line
-First, try building each target at the command line.
+<div class="primer-spec-callout info" markdown="1">
+**Pro-tip:** In future projects, you'll have to figure out which files to include in a build.  Generally, each time you see `#include "myfile.h"`, you will include a corresponding `myfile.cpp` file in the build.  Be sure that you include only one `main()` function.
```

Comment:
May be worth adding "Never add `.h` files to the build target." or something like that.

Reply:
Good idea.